test/e2e/images/images.go

```diff
@@ -18,7 +18,7 @@ func init() {
 
 	netcat = os.Getenv("NETCAT_IMAGE")
 	if netcat == "" {
-		netcat = "openshift/origin-network-tools:latest"
+		netcat = "ingressnodefirewall/netcat:latest"
```

Comment:
will this CI dev-script be able to pull this for E2E OCP CI ?

Reply:
It should